zc_common/jwt_auth/permissions.py

```diff
@@ -63,3 +63,8 @@ def has_update_permission(self, request, view):
 class EventViewPermission(BasePermission):
     def has_create_permission(self, request, view):
         return is_service(request)
+
+
+class IsStaffPermission(permissions.BasePermission):
```

Comment:
Why are we adding this permission class here? if this is a good place for it, why we don't have other permission classes like User or service here as well?